connector/connect/server/src/main/scala/org/apache/spark/sql/connect/artifact/SparkConnectArtifactManager.scala

```diff
@@ -157,10 +159,34 @@ class SparkConnectArtifactManager private[connect] {
       }
     }
   }
+
+  private[connect] def uploadArtifactToFs(
+      sessionHolder: SessionHolder,
+      remoteRelativePath: Path,
+      serverLocalStagingPath: Path): Unit = {
+    val hadoopConf = sessionHolder.session.sparkContext.hadoopConfiguration
+    assert(remoteRelativePath.startsWith(
+      SparkConnectArtifactManager.forwardToFSPrefix + File.separator)
```

Comment:
I am fine with this but one thing is that now here we pass an absolute path with the prefix. Except here, all other places pass a relative path around. Is that fine, @hvanhovell and @vicennial?

Reply:
In the design suggested by @vicennial , We defined the artifact path to be `forward_to_fs/{dest_fs_path}`, and `dest_fs_path` is absolute path to cloud storage. I think it is fine and it works in all cases.

Reply:
🆗